src/local-ssh/proxy.ts

```diff
@@ -34,46 +46,6 @@ if (!options) {
     process.exit(1);
 }
 
-import { NopeLogger, DebugLogger } from './logger';
-const logService = options.debug ? new DebugLogger(path.join(os.tmpdir(), `lssh-${options.host}.log`)) : new NopeLogger();
-
-import { TelemetryService } from './telemetryService';
-const telemetryService = new TelemetryService(
-    process.env.SEGMENT_KEY!,
-    options.machineID,
-    process.env.EXT_NAME!,
-    process.env.EXT_VERSION!,
-    options.gitpodHost,
-    logService
-);
-
-const flow: SSHUserFlowTelemetry = {
-    flow: 'local_ssh',
-    gitpodHost: options.gitpodHost,
-    workspaceId: '',
-    processId: process.pid,
-};
-
-telemetryService.sendUserFlowStatus('started', flow);
```

Comment:
it is bad that it is not first event anymore, since we cannot see how much time it takes from started to connected, i.e. we cannot tell effect of reading versions on connection performance. Maybe we omit versions then for started?